sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/DoNotCatchNullReferenceException.cs

```diff
@@ -0,0 +1,20 @@
+using System;
```

Comment:
You're probably right, it's an edge case. So just add a test. Perhaps also add a test with nothing but throw (and throw e;)?